### ISSUE TITLE
no webGL no serial available but function tested ok otherwise

### ISSUE THREAD

**Issue by diplodocuslongus** [adafruit/Adafruit_WebSerial_3DModelViewer#4]
Hello,
I'm on ubuntu 18.04, latest available chrome installed (v99), _webserial_ enabled and tested successfully with 
https://googlechromelabs.github.io/serial-terminal/
and _webgl_ enabled and tested ok with
https://get.webgl.org/
(I see the rotating cube).
I am using your IMU Fusion Breakout - BNO055 (Product 2472) with an arduino uno, I can read orientation data via the serial monitor using the provided library examples.

But when I open the page at https://adafruit.github.io/Adafruit_WebSerial_3DModelViewer/ I get:
```
Sorry, Web Serial is not supported on this device, make sure you're running Chrome 78 or later and have enabled the #enable-experimental-web-platform-features flag in chrome://flags
Sorry, WebGL is not supported on this device.
```
Why is it so?

**Comment by makermelissa** [comment]
I think this was fixed by #5 if you'd like to try again.

**Comment by diplodocuslongus** [comment]
Yes indeed Melissa, works right out of the box, thanks!
